CMakeLists.txt

```diff
@@ -387,29 +387,26 @@ endforeach()
 
 # Find Lua interpreter
 include(LuaHelpers)
-set(LUA_DEPENDENCIES lpeg mpack bit)
 if(NOT LUA_PRG)
   foreach(CURRENT_LUA_PRG luajit lua5.1 lua5.2 lua)
     # If LUA_PRG is set find_program() will not search
     unset(LUA_PRG CACHE)
-    unset(LUA_PRG_WORKS)
     find_program(LUA_PRG ${CURRENT_LUA_PRG})
 
     if(LUA_PRG)
-      check_lua_deps(${LUA_PRG} "${LUA_DEPENDENCIES}" LUA_PRG_WORKS)
```

Comment:
Did it work that way before? Directly below this line there is a check for `LUA_PRG_WORKS` that breaks out of the `foreach` loop. It does this on the first pass of the loop for which both of the following conditions are true:

* a suitable lua interpreter has been found
* that suitable lua interpreter has the required modules installed

I'm willing to fix that bug also, but it will have to wait until tonight (GMT-8)